pages.de/windows/dir.md

```diff
@@ -0,0 +1,28 @@
+# dir
+
+> Zeigt den Inhalt von Arbeitsverzeichnisses an.
+> Weitere Informationen: <https://docs.microsoft.com/de-de/windows-server/administration/windows-commands/dir>.
+
+- Zeigt den Inhalt des aktuellen Arbeitsverzeichnisses an:
+
+`dir`
+
+- Zeigt den Inhalt des angegebenen Arbeitsverzeichnisses an:
+
+`dir {{pfad/zu/verzeichnis}}`
+
+- Zeigt den Inhalt (inklusive versteckte Inhalte) des aktuellen Arbeitsverzeichnisses an:
+
+`dir /A`
+
+- Zeigt den Inhalt (inklusive versteckte Inhalte) des angegebenen Arbeitsverzeichnisses an:
+
+`dir {{pfad/zu/verzeichnis}} /A`
+
+- Zeigt den Inhalt mit Dateibesitzinformationen des aktuellen Arbeitsverzeichnisses an:
+
+`dir /q`
+
+- Zeigt den Inhalt mit Dateibesitzinformationen des angegebenen Arbeitsverzeichnisses an:
+
+`dir {{pfad/zu/verzeichnis}} /q`
```

Comment:
We only accept translations of the main pages, to keep this simple to maintain. You can add  these examples to the English page if you want.